File: AnkietyUW/src/AnkietyUW.Contracts/TestDto/DataTransferObjects/CreateTestDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AnkietyUW.Contracts.TestDto.DataTransferObjects
{
    public class CreateTestDto
    {
        [Required]
        public int FirstQuestionAddSeconds { get; set; }
        [Required]
        public int SecondQuestionAddSeconds { get; set; }
        [Required]
        public int ThirdQuestionAddSeconds { get; set; }
        [Required]
        public int FourthQuestionAddSeconds { get; set; }
        [Required]
        public int CompletedSeriesCounter { get; set; }
        [Required]
        public ICollection<TestTimeDto> TestTimes { get; set; }
        [Required]
        public int TimeToFillTestAddSeconds { get; set; }
    }
}

File: AnkietyUW/src/AnkietyUW.Frontend/src/models/createTestDto.ts
import {TestTimeDto} from "./testTimeDto";

export class CreateTestDto{
    firstQuestionAddSeconds: number;
    secondQuestionAddSeconds: number;
    thirdQuestionAddSeconds: number;
    fourthQuestionAddSeconds: number;
    testTimes: TestTimeDto[];
    timeToFillTestAddSeconds: number;
}